### PR TITLE
Update Parameter Name from `tikv-importer.incremental-import` to `tikv-importer.parallel-import`

### PR DESCRIPTION
**Pull request by lyzx2001**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Update Parameter Name from `tikv-importer.incremental-import` to `tikv-importer.parallel-import`.

ref https://github.com/pingcap/tidb/pull/45539

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/14350
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by ran-huang — APPROVED** [review]

File: releases/release-5.4.0.md
Comment (ran-huang) [inline releases/release-5.4.0.md:1]: 不应该改以前的 release notes 吧，以前的版本还是旧的 config name

Comment (qiancai) [reply releases/release-5.4.0.md:1]: +1

Comment (lyzx2001) [reply releases/release-5.4.0.md:1]: 已删除 `releases/release-5.4.0.md` 中的改动

**Review by lance6716 — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Comment by qiancai** [comment]
@lyzx2001 请看看这里需要更改吗
image

**Comment by lyzx2001** [comment]
> @lyzx2001 请看看这里需要更改吗
> 
> https://docs.pingcap.com/tidb/dev/tidb-lightning-configuration#tidb-lightning-task image

已更改

**Review by ran-huang — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ driver = "file"
 # 注意，该参数**不是用于增量导入数据**，仅限目标表为空的场景使用。
```
Comment (qiancai) [inline tidb-lightning/tidb-lightning-configuration.md:118]:
```suggestion
# TiDB Lightning 并行导入参数，仅限目标表为空的场景使用。
```

Comment (lyzx2001) [reply tidb-lightning/tidb-lightning-configuration.md]: 已修改

**Pull request merged**